spec/models/survey_spec.rb

```diff
@@ -1,21 +1,28 @@
 require 'rails_helper'
 
+# TODO: use FactoryBot for entties here
+
 RSpec.describe Survey, type: :model do
   it 'is valid with valid attributes' do
-    survey = Survey.new(question: 'Is this thing works?', start_date: Date.today, end_date: Date.tomorrow)
-    survey.survey_answers.new(answer: 'yes')
-    survey.survey_answers.new(answer: 'no')
+    user = FactoryBot.build(:user, :admin)
+    survey = user.surveys.new(FactoryBot.attributes_for(:survey))
+
+    2.times do
+      survey.survey_answers.new(FactoryBot.attributes_for(:survey_answer))
+    end
```

Comment:
it could make sense to have:
```ruby
it 'persisted' do
   question = 'what number?'
   survey = described_class.new(
      user: user,
      question: question
      surway_answers: [
         FactoryBot.build(:surway_answer, answer: '1'), 
         FactoryBot.build(:surway_answer, answer: '2')
      ]
   )
   expect(survey.save).to be_truthy
   expect(survey.user_id).to eq(user.id)
   expect(survey.question).to eq(question)
   expect(survey.surwey_answers.pluck(:answer).sort).to eq(%w[1 2])
end
```